src/recordlinker/linking/clean.py

```diff
@@ -0,0 +1,122 @@
+"""
+recordlinker.linking.clean
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This module is used to clean the data before running the linkage algorithm
```

Comment:
Nit: when we change the module name, I would also alter the copy of this comment to reflect that it's concerned with removing skip-values, until/unless we put more cleaning functions in